validators/terravaloper1qrgwphlf9c64m5ys6pzsvy055ud03e4kr7sq8u/profile.json

```diff
@@ -0,0 +1,10 @@
+{
+  "contact": {
+    "email": "lunc_nymph@proton.me",
+    "keybase": "https://keybase.io/lunc_nymph",
+    "twitter": "https://twitter.com/lunc_nymph",
```

Comment:
Trailing commas are not allowed on JSON.  If you remove this, you should be ready to go!

Reply:
sorry, changed. thanks